docs/release-notes/index.md

```diff
@@ -0,0 +1,1243 @@
+---
+navigation_title: "Elastic APM Java Agent"
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/java/current/release-notes-1.x.html
+  - https://www.elastic.co/guide/en/apm/agent/java/current/release-notes.html
+---
+
+# Elastic APM Java Agent release notes [elastic-apm-java-agent-release-notes]
+
+Review the changes, fixes, and more in each version of Elastic APM Java Agent.
+
+To check for security updates, go to [Security announcements for the Elastic stack](https://discuss.elastic.co/c/announcements/security-announcements/31).
+
+% Release notes includes only features, enhancements, and fixes. Add breaking changes, deprecations, and known issues to the applicable release notes sections.
+
+% ## version.next [elastic-apm-java-agent-versionext-release-notes]
+% **Release date:** Month day, year
+
+% ### Features and enhancements [elastic-apm-java-agent-versionext-features-enhancements]
+
+% ### Fixes [elastic-apm-java-agent-versionext-fixes]
+
+## 1.52.1 [elastic-apm-java-agent-1521-release-notes]
```

Comment:
Because I haven't seen that before: is the name in the brackets (`[elastic-apm-java-agent-1521-release-notes]`) some kind of anchor? Do you have a link to the documentation of this feature?

I think turning the version number into a string without separators could lead to collisions and therefore is a bad idea:
E.g. `1.52.1` and `1.5.21` both result in the same `1520` string. Any reason not to be safe and go with `elastic-apm-java-agent-1-52-1-release-notes`?

Reply:
>Because I haven't seen that before: is the name in the brackets (`[elastic-apm-java-agent-1521-release-notes]`) some kind of anchor? Do you have a link to the documentation of this feature?

Yes. The documentation is [here](https://elastic.github.io/docs-builder/syntax/headings/#custom-anchor-links) (and I just opened a [PR](https://github.com/elastic/docs-builder/pull/677) to clarify the example).

>I think turning the version number into a string without separators could lead to collisions and therefore is a bad idea:

The syntax used in the IDs was automated, but we can manually update them here.

Reply:
I pushed an update in https://github.com/elastic/apm-agent-java/pull/3981/commits/319f9b17071a091f5e8c34960772b8888407f0e3.